artifactory/cli.go

```diff
@@ -658,6 +658,14 @@ func getUploadFlags() []cli.Flag {
 			Name:  "symlinks",
 			Usage: "[Default: false] Set to true to preserve symbolic links structure in Artifactory.` `",
 		},
+		cli.StringFlag{
+			Name:  "sync-delete",
+			Usage: "[Optional] Specific path to sync after upload. This path will eventually contain only the artifacts uploaded during this upload operation.` `",
+		},
+		cli.BoolFlag{
+			Name:  "quiet",
+			Usage: "[Default: false] Set to true to skip the sync-delete confirmation message.` `",
```

Comment:
Since this "quiet" option is used by multiple commands already, I suggest having a function creating this option, so that the code creating it can be reused.
See the "getFailNoOpFlag()" function as an example.